drivers/storage/portworx/components.go

```diff
@@ -377,6 +396,230 @@ func createVolumePlacementStrategyCRD() error {
 	return k8s.Instance().ValidateCRD(resource, 1*time.Minute, 5*time.Second)
 }
 
+func (p *portworx) createPortworxStorageClasses(t *template) error {
+	ownerRef := metav1.NewControllerRef(t.cluster, controllerKind)
+	docAnnotations := map[string]string{
+		"params/docs":              "https://docs.portworx.com/scheduler/kubernetes/dynamic-provisioning.html",
+		"params/fs":                "Filesystem to be laid out: none|xfs|ext4",
+		"params/block_size":        "Block size",
+		"params/repl":              "Replication factor for the volume: 1|2|3",
+		"params/secure":            "Flag to create an encrypted volume: true|false",
+		"params/shared":            "Flag to create a globally shared namespace volume which can be used by multiple pods: true|false",
+		"params/priority_io":       "IO Priority: low|medium|high",
+		"params/io_profile":        "IO Profile can be used to override the I/O algorithm Portworx uses for the volumes: db|sequential|random|cms",
+		"params/group":             "The group a volume should belong to. Portworx will restrict replication sets of volumes of the same group on different nodes. If the force group option 'fg' is set to true, the volume group rule will be strictly enforced. By default, it's not strictly enforced",
+		"params/fg":                "This option enforces volume group policy. If a volume belonging to a group cannot find nodes for it's replication sets which don't have other volumes of same group, the volume creation will fail",
+		"params/label":             "List of comma-separated name=value pairs to apply to the Portworx volume",
+		"params/nodes":             "Comma-separated Portworx Node ID's to use for replication sets of the volume",
```

Comment:
remove this too.

For labels, we automatically pick k8s pvc labels now

For nodes, we want users to use volume placement strategies.